examples/json/src/tests.rs

```diff
@@ -5,18 +5,16 @@ use rocket::http::{Status, ContentType};
 use rocket::Response;
 
 macro_rules! run_test {
-    ($req:expr, $test_fn:expr) => ({
-        let rocket = rocket::ignite()
-            .mount("/message", routes![super::new, super::update, super::get])
-            .catch(errors![super::not_found]);
-
+    ($req:expr, $test_fn:expr, $rocket:expr) => ({
```

Comment:
Can we make `rocket` the first argument to this macro? It looks a bit odd hanging off at the end there.